package.json

```diff
@@ -8,6 +8,7 @@
     "@rails/ujs": "^6.0.0",
     "@rails/webpacker": "5.4.3",
     "bootstrap": "^5.1.3",
+    "bootstrap-icons": "^1.8.3",
```

Comment:
added but not used? seemed like a good idea? :)

Reply:
🙈 our bad!

Reply:
now we're using this library keeping our code clean and easy to read! 🤓